src/org/parosproxy/paros/extension/ExtensionHookMenu.java

```diff
@@ -331,6 +344,14 @@ public void addToolsMenuItem(ZapMenuItem menuItem) {
         addToolsMenuItemImpl(menuItem);
     }
 
+    public void addImportMenuItem(ZapMenuItem menuItem) { addImportMenuItemImpl(menuItem); }

```

Comment:
Yes this is syntactically correct but it would be nice for things to be consistent. Opening curly brace on function declaration, code, closing curly brace on a separate line.

Reply:
Same for `getImport()` above...

Reply:
Just now I checked it, these functions were formatted in one line by default Intellij editor. Will do all the changes @kingthorin

Reply:
No worries, every IDE does different stuff. Thanks!